src/devices/inovelli.ts

```diff
@@ -2167,7 +1974,19 @@ const definitions: DefinitionWithExtend[] = [
         vendor: 'Inovelli',
         description: '2-in-1 switch + dimmer',
         exposes: exposesListVZM31.concat(identify().exposes as Expose[]),
-        extend: [inovelliExtend.addCustomClusterInovelli()],
+        extend: [
+            deviceEndpoints({
+                endpoints: {'1': 1, '2': 2, '3': 3},
+                multiEndpointSkip: ['state', 'power', 'energy', 'brightness'],
+            }),
+            inovelliExtend.addCustomClusterInovelli(),
+            electricityMeter({
+                current: false,
+                voltage: false,
+                power: {min: 15, max: 3600, change: 1},
```

Comment:
The min/max/change here doesn't seem to be respected (I'm not sure why). When I run a reconfigure, it's added with `activePower` min: 10, max: 65000, change: 5 and `currentSummDelivered` min: 10, max: 65000, change: 0.1 instead of what's provided here (matching the custom call below)

Reply:
There are two reporting mechanisms in the device that can sometimes overlap. When we first created the device we took our configuration options from our Z-Wave switch not knowing (at the time) that Zigbee had its own built in energy / power reporting functionality. So, you can have different configurations in the exposes tab vs the reporting tab. If I could do it again, I would remove config options 17-19 and only used the built in Zigbee reporting. Anyway, I think that is why you may be seeing this. Try setting P17-19 to 0 and then test the reporting configuration.

Reply:
Sorry, I think my original message wasn't very clear.

I haven't changed any of the reporting parameters in the exposes tab. When the original code is running (see single commit diff: https://github.com/Koenkk/zigbee-herdsman-converters/pull/8402/commits/44b0a9afe73d97ce2acb0a59410ee4a2c1a292c4), it explicitly sets (manually) the reporting to the min/max/change values as described on line 2025 and 2026. This leads to the following screenshot:

![image](https://github.com/user-attachments/assets/86dc0296-0d35-4928-a0ca-c3bd4dae1d18)


I switched the code to use the modern extend instead and passed it along the same min/max/change values as before. However, this doesn't seem to be taking affect. Leading to the following screenshot:

```
electricityMeter({
    current: false,
    voltage: false,
    power: {min: 15, max: 3600, change: 1},
    energy: {min: 15, max: 3600, change: 0},
})
```
![image](https://github.com/user-attachments/assets/a216411e-1e6c-4957-8db9-6854425b0698)

I see duplication of the reporting calls (but with the correct values). Is that expected? 

I think this is more of a question for @Koenkk.

Reply:
Oh yeah, that isn't the question I thought it was and would be better answered by @Koenkk lol.